rpc-jobs/jjb-setup.yml

```diff
@@ -1,4 +1,24 @@
 - job:
+    name: 'Merge-Trigger-JJB'
+    project-type: freestyle
```

Comment:
We can merge this as a freestyle and I can work on updating it to a pipeline!

The JJB failures was because there was an old job named Merge-Trigger-JJB that was a workflow job, so it failed when it tried to update the job as a freestyle one. I renamed the old one to Merge-Trigger-JJB-pipeline and it ran successfully.